scripts/app

```diff
@@ -99,11 +99,19 @@ compose() {
 }
 
 update_installed_apps() {
+    if [[ ! -f "${USER_FILE}.lock" ]]; then
```

Comment:
This is checking if the lock file does not exist, you want to check if it does exist.
```suggestion
    if [[ -f "${USER_FILE}.lock" ]]; then
```